src/EditorFeatures/CSharpTest/EditAndContinue/LineEditTests.cs

```diff
@@ -1417,6 +1417,50 @@ class C
                 Array.Empty<string>());
         }
 
+        [Fact]
+        public void Event_ExpressionBodyToBlock()
+        {
```

Comment:
ok. thanks!

---
In reply to: [128395783](https://github.com/dotnet/roslyn/pull/20931#discussion_r128395783) [](ancestors = 128395783,128395722)